models/evms/evms_info.sql

```diff
@@ -0,0 +1,21 @@
+{{ config(
+        tags = ['dunesql', 'static'],
+        alias = alias('info', timestamp ),
+        post_hook='{{ expose_spells(\'["ethereum", "polygon", "bnb", "avalanche_c", "gnosis", "fantom", "optimism", "arbitrum", "celo"]\',
+                                    "sector",
+                                    "evms",
+                                    \'["hildobby"]\') }}')
+}}
+
+SELECT chain_id, blockchain, name, chain_type, rollup_type, native_token_symbol, wrapped_native_token_address, explorer_link, first_block_time
+FROM (VALUES
+        (1, 'ethereum', 'Ethereum', 'Layer 1', NULL, 'ETH', 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2, 'https://etherscan.io', timestamp '2015-07-30 15:26')
+        , (43114, 'avalanche_c', 'Avalanche', 'Layer 1', NULL, 'AVAX', 0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7, 'https://optimistic.etherscan.io', timestamp '2020-09-23 11:02')
+        , (42161, 'arbitrum', 'Arbitrum', 'Layer 2', 'Optimistic Rollup', 'ETH', 0x82af49447d8a07e3bd95bd0d56f35241523fbab1, 'https://arbiscan.io', timestamp '2021-05-29 00:35')
+        , (10, 'optimism', 'Optimism', 'Layer 2', 'Optimistic Rollup', 'ETH', 0x4200000000000000000000000000000000000006, 'https://optimistic.etherscan.io', timestamp '2021-11-11 21:16')
```

Comment:
OP Mainnet

Reply:
Another comment - the legacy OVM1 started July 2021, but the current system is Nov 11 2021.

Maybe we add a row for optimism_ovm1_legacy ?